crates/bevy_animation/src/util.rs

```diff
@@ -0,0 +1,10 @@
+/// Steps between two different discrete values of any clonable type.
```

Comment:
I recommend being more precise with the wording here. The function does not clone anything nor does it require the inputs to be T: Clone.